### ISSUE TITLE
Posts from the WordPress App Don't Trigger an Subscription Update

### ISSUE THREAD

**Issue by mrjarbenne** [hwdsb/subscribe-by-email#7]
When posts are added from the iOS app, rather than from the web interface, they don't result in an email. I've confirmed this glitch when the settings are configured to send an email immediately. I've switched the drop down to a daily digest to see if the post gets included in that digest (digest will be sent out tomorrow.)

**Comment by r-a-y** [comment]
The WordPress iOS app uses XMLRPC to authenticate and publish posts.

On my local rig, I tested publishing a post with Open Live Writer, a Windows blogging client utilizing XMLRPC, and that was able to trigger the immediate subscription email.

I don't have access to an iOS device, but I can test Android on the Sandbox site once your daily digest test is over tomorrow.

**Comment by mrjarbenne** [comment]
Daily Digest didn't include the post from the iOS app. Let me know if you have better luck with Android.

**Comment by r-a-y** [comment]
I was able to duplicate the problem in the Android app.

If I comment out the following line:
https://github.com/hwdsb/subscribe-by-email/blob/4c12937d00bca454562ac558500c731e49ba6e3b/subscribe-by-email.php#L713

Posting from the Android app works, but it looks like I added this portion of the code for Gutenberg and scheduled posts. I'm going to do some more testing with the block editor before committing a fix.
